src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <div class="card__container-image">
+        <img
+        src="images/imac.jpeg"
+        alt="imac_image"
+        class="card__image"
+        >
+      </div>
+
+      <div class="card__container-card-name">
+        <h1 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      </div>
+
+      <div class="card__container-product-code">
+        <h3 class="card__product-code">Product code: 195434</h3>
+      </div>
+
+      <div class="card__container-stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <h3 class="card__reviews-number">Reviews: 5</h3>
+      </div>
+
+      <div class="card__container-price">
+        <h3 class="card__price">Price:</h3>
+
+        <h1 class="card__price-number">$2,199</h1>
+      </div>
+
+      <a
+        class="card__link"
+        href="#buy"
+        data-qa="hover"
+      >
+        <div class="card__buy-button">Buy</div>
+      </a>
```

Comment:
```suggestion
      <a
        class="card__link"
        href="#buy"
        data-qa="hover"
      >
        Buy
      </a>
```
Can't you just put text instead of nesting div into a?